src/bootstrap-table.js

```diff
@@ -1105,7 +1105,6 @@ class BootstrapTable {
       if (this.options.sortReset) {
         this.unsortedData = [...this.data]
       }
-
```

Comment:
This line does not need to be deleted.

Reply:
Sorry about that. Was definitely unintentional! I re-added the line.

Reply:
Sorry about that. Was definitely unintentional! I re-added the line.